src/azul/plugins/metadata/hca/transform.py

```diff
@@ -138,32 +155,28 @@ def to_index(self, value: Optional[JSON]) -> str:
         Traceback (most recent call last):
         ...
         azul.RequirementError: The `unit` entry must be a string
-
         """
         if value is None:
             return NullableString.null_string
         else:
+            msg = 'A dictionary with entries for `value` and `unit` is required'
+            require(len(value) == 2, msg)
```

Comment:
Move to else branch of try.

Reply:
It proved challenging to check the length of the incoming `value` after the resignation of the variable. I refactored it to `values` and moved the check to the else branch of the try block as requested.